Dockerfile

```diff
@@ -1 +1,79 @@
 FROM 345280441424.dkr.ecr.ap-south-1.amazonaws.com/ark_base_java8:latest
+LABEL ORG="Armedia LLC" \
+      VERSION="1.0" \
+      IMAGE_SOURCE=https://github.com/ArkCase/ark_arkcase_core \
+      MAINTAINER="Armedia LLC"
+
+ARG ARKCASE_VERSION=2021.03-RC11
+ARG TOMCAT_VERSION=9.0.50 
+ARG TOMCAT_MAJOR_VERSION=9
+ARG SYMMENTRIC_KEY=9999999999999999999999
```

Comment:
typo, should be `SYMMETRIC_KEY`

Reply:
Can you take all these version numbers from the environment somehow or do they have to be hardcoded in the Dockerfile?

Reply:
removed hardcoding of version details in the dockerfile.